static/js/components/moving_object/MovingObjectObsPlanPage.jsx

```diff
@@ -0,0 +1,293 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Paper from "@mui/material/Paper";
+import Grid from "@mui/material/Grid";
+import { makeStyles } from "@mui/styles";
+import Form from "@rjsf/mui";
+import validator from "@rjsf/validator-ajv8";
+import { showNotification } from "baselayer/components/Notifications";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import TextLoop from "react-text-loop";
+import Dialog from "@mui/material/Dialog";
+import DialogContent from "@mui/material/DialogContent";
+import CircularProgress from "@mui/material/CircularProgress";
+import { MyObjectFieldTemplate } from "../gcn/GcnSelectionForm";
+
+import * as movingObjectActions from "../../ducks/moving_object";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    backgroundColor: theme.palette.background.paper,
+    whiteSpace: "pre-line",
+  },
+  paperContent: {
+    padding: "1rem",
+    marginBottom: "1rem",
+  },
+  spinner: {
+    margin: "auto",
+    fontWeight: "bold",
+    fontSize: "1.25rem",
+    textAlign: "center",
+  },
+}));
+
+const PlaceHolder = () => {
+  const classes = useStyles();
+  return (
+    <div className={classes.spinner}>
+      <TextLoop interval={1500}>
```

Comment:
We should use this more.

Reply:
yeah! I stumbled upon it when fixing some code in the finding chart stuff, it's nice to give the user a loading bar that gives them the illusion that something is happening, to capture their attention. Makes them forget that some things are actually quite slow 😉